finetune/util/metrics.py

```diff
@@ -35,7 +35,7 @@ def sequence_labeling_token_counts(true, predicted):
     Return FP, FN, and TP counts
     """
 
-    unique_classes = set([seq['label'] for seqs in true for seq in seqs])
+    unique_classes = set([seq['label'] for seqs in true + predicted for seq in seqs])
```

Comment:
Hehehe this is totally unrelated but this has bitten me before so I'm glad this is going in.